### ISSUE TITLE
2 tags that match pattern on last commit produces NumberFormatException: For input string: ""

### ISSUE THREAD

**Issue by rohanmartin** [gladed/gradle-android-git-version#43]
androidGitVersion version 0.4.0

with build.gradle:
```
    prefix ''
    tagPattern "${prefix}v[0-9]+.*"
```

Run command line:
```
git tag -a v1.0.0-rc.1 -m "v1.0.0 RC1"
git tag -a v1.0.0 -m "v1.0.0"
./gradlew --stacktrace androidGitVersion
```

Should give a version based on the most recent tag (v1.0.0), but:

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/User/Documents/workspace/Project/app/build.gradle' line: 27

* What went wrong:
A problem occurred evaluating project ':app'.
> For input string: ""

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
...
> Caused by: java.lang.NumberFormatException: For input string: ""
        at com.gladed.gradle.androidgitversion.AndroidGitVersionExtension$_scan_closure9$_closure13.doCall(AndroidGitVersion.groovy:249)
        at com.gladed.gradle.androidgitversion.AndroidGitVersionExtension$_scan_closure9.doCall(AndroidGitVersion.groovy:249)
        at com.gladed.gradle.androidgitversion.AndroidGitVersionExtension.scan(AndroidGitVersion.groovy:245)
        at com.gladed.gradle.androidgitversion.AndroidGitVersionExtension.code(AndroidGitVersion.groovy:160)
        at com.gladed.gradle.androidgitversion.AndroidGitVersionExtension$code.call(Unknown Source)
        at build_8e0jag8zpised42jud3iwyaxc$_run_closure2$_closure4.doCall(/Users/User/Documents/workspace/Project/app/build.gradle:27)
...

**Comment by gladed** [comment]
Thanks for submitting this. Expect a patch soon.

As a temporary workaround, you can use `prefix 'v'`. And if you want `v` in the resulting name you can customize `format` to include it.

**Comment by gladed** [comment]
Fixed in `0.4.1`

**Comment by gladed** [comment]
@rohanmartin please let me know if this doesn't address the issue! BTW the latest version is now `0.4.3`.
